docs/examples/ollama/model.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: llmaz.io/v1alpha1
+kind: OpenModel
+metadata:
+  name: qwen2-0--5b
+spec:
+  familyName: qwen2
+  source:
+    uri: OLLAMA://qwen2:0.5b
```

Comment:
Let's use lower case for consistency, just like http, although we'll convert to upper case for comparison in the runtime.